presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunner.java

```diff
@@ -0,0 +1,301 @@
+    public NodePartitioningManager getNodePartitioningManager()
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ConnectorPlanOptimizerManager getPlanOptimizerManager()
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public StatsCalculator getStatsCalculator()
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public Optional<EventListener> getEventListener()
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public TestingAccessControlManager getAccessControl()
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public MaterializedResult execute(String sql)
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public MaterializedResult execute(Session session, String sql, List<? extends Type> resultTypes)
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public List<QualifiedObjectName> listTables(Session session, String catalog, String schema)
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public boolean tableExists(Session session, String table)
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public void installPlugin(Plugin plugin)
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public void createCatalog(String catalogName, String connectorName, Map<String, String> properties)
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public void loadFunctionNamespaceManager(String functionNamespaceManagerName, String catalogName, Map<String, String> properties)
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public Lock getExclusiveLock()
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public int getNodeCount()
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public Session getDefaultSession()
+    {
+        return null;
+    }
+
+    @Override
+    public MaterializedResult execute(Session session, String sql)
+    {
+        String[] command = {
+                "/opt/presto-cli",
+                "--server",
+                "presto-coordinator:" + coordinatorPort,
+                "--catalog",
+                catalog,
+                "--schema",
+                schema,
+                "--execute",
+                sql
+        };
+
+        Container.ExecResult execResult = null;
+        try {
+            execResult = coordinator.execInContainer(command);
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed with error message: " + e.getMessage());
+        }
+        catch (InterruptedException e) {
+            fail("Presto CLI failed with error message: " + e.getMessage());
```

Comment:
It's clearer to the end user to let the exception bubble up and fail the test than convert it into a fail call.

Reply:
updated the code.